src/main/resources/application.yaml

```diff
@@ -21,4 +21,18 @@ quarkus:
       format: "%d{HH:mm:ss} %-5p [%c{2.}] (%t) %s%e%n"
       rotation:
         max-backup-index: 5
-        max-file-size: 10M
\ No newline at end of file
+        max-file-size: 10M
+
+honeycomb:
+    enabled: false
+    dataset: indy-sidecar-test
+    write-key: test
+    base-sample-rate: 20
+    functions: |-
+      /api/content/.*|sidecar.content|20,
+      /api/folo/admin/.*|sidecar.folo.admin|1,
+      /api/folo/track/.*|sidecar.folo.content|1,
```

Comment:
In sidecar, we only use folo/track. Other functions are only tracked on server side.

Reply:
@ruhan1 , maybe also include content/* ? It serves the content from the historical archive if existed...
This is inherited from your gateway so easy to configure.

Reply:
atm let's keep content too. actually pnc builder download/upload requests are always have folo/track. but it might change when sidecar comes into place.